version/src/main/java/org/jboss/as/version/ProductConfig.java

```diff
@@ -150,8 +150,8 @@ public ProductConfig(final String productName, final String productVersion, fina
         this.name = productName;
         this.version = productVersion;
         this.consoleSlot = consoleSlot;
-        this.defaultStability = Stability.DEFAULT;
-        this.stabilities = EnumSet.of(this.defaultStability);
+        this.defaultStability = Stability.COMMUNITY;
+        this.stabilities =  Collections.unmodifiableSet(EnumSet.range(Stability.DEFAULT, Stability.COMMUNITY));
```

Comment:
I would keep this logic internal to the Stability object.
e.g.
```java
this.stabilities = EnumSet.allOf(Stability.class).stream().filter(this.defaultStability::enables).collect(Collectors.toUnmodifiableSet());
```

Reply:
@pferraro thanks for reviewing, done

Since that was the only pending fix, I am going to move on with it so I can prepare a release today

Reply:
Well, wait .... now I am wondering, I would expect to see the mixed domain integration tests failing with this PR in, since we will be assuming EAP 7.4 legacy host will be running in "default" stability level, hence the mixed domains should have failed because WildFly will be running by default in "community" stability level.

Looking ....

Reply:
Ok, got it, I've realized today that full integration Jobs are ignoring the mixed domain tests because of a misconfiguration of those two Jobs (they are using JDK 17 but legacy JDK to launch the legacy servers is not configured). I'll fix them once we have integrated both this PR and the WildFly counterpart, so we can continue without breaking WildFly Core CI until the WildFly full gets merged.

Reply:
WildFly Core integration Jobs have been fixed to include the expected JDK to be able to run the mixed domain tests